carla_integration/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+#  Copyright (C) 2018-2020 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# Docker Compose Spec Version
+version: '2'
+
+services:
+  roscore:
+    image: usdotfhwastol/carma-base:CARMASystem_3.3.0
+    network_mode: host
+    container_name: roscore
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.0.100
+    volumes:
+      - /opt/carma/.ros:/home/carma/.ros
+    restart: always
+    command: roscore
+
+  platform:
+    image: usdotfhwastol/carma-platform:CARMASystem_3.3.0
+    network_mode: host
+    container_name: platform
+    volumes_from:
+      - container:carma-config:ro
+    environment:
+      - ROS_IP=192.168.0.100
+      - NVIDIA_VISIBLE_DEVICES=all
+    volumes:
+      - /opt/carma/logs:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/calibration:/opt/carma/vehicle/calibration
+      - /opt/carma/maps:/opt/carma/maps
+      - /opt/carma/routes:/opt/carma/routes
+      - /opt/carma/yolo:/opt/carma/yolo
+    command: wait-for-it.sh localhost:11311 -- roslaunch /opt/carma/vehicle/config/carma_docker.launch
```

Comment:
Since you are building a docker file for the simulation components that file should be launched here using the drivers.launch file in the same manner as the other drivers. Then you don't need to modify the top level launch file.

Reply:
Added the carla_ackermann_control component